### PR TITLE
MHV-64025-list-count-per-filter

### PR DESCRIPTION
**Pull request by agravell047**
## Summary

- Adds a count object to the metadata returned from the API

## Related issue(s)

- https://jira.devops.va.gov/browse/MHV-64025
Screenshot 2024-11-21 at 9 42 30 AM

## Testing done

- Tested locally
- Added test

## Screenshots
N/A

## What areas of the site does it impact?
MHV medications app

### CODE REVIEW

**Review by aherzberg — APPROVED** [review]
LGTM

File: modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb [hunk 1]
```diff
@@ -138,6 +160,36 @@ def collection_resource
           client.get_active_rxs_with_details
         end
       end
+
+      def set_filter_metadata(list)
```
Comment (robertbylight) [inline modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb:164]: just leaving this comment here, we already talked about moving this function `set_filter_metadata()` and the ones associated to it to the helper file, however we had some problems trying that, given our time restraint, well move it on a separate PR.

**Review by rjohnson2011 — APPROVED** [review]
